javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
+  if (useFormalName) {
+  return ("Lord" + " " + fullname1 + " " + fullname2)
+} else {
+  return (fullname1 + " " + fullname2)
+}
+}
+getFullname(fullname1, fullname2);
+/* the getFullname for women will work the similar way: we need to create boolean variable useWomenName and if it's true
+we should use Queen as a formal way instead of Lord.
+*/
+
+// Event application
+function getEventWeekday(){
+let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const todaysDate = new Date();
+
+}
```

Comment:
You had a good idea! You just did not finish the function! The function is really easy to complete: 

1. First, you need a parameter for the number of days we have to look in the future, for example `daysInTheFuture`.
1. The `Date` object you create in line 23 gives you an object that represents today's date. This object also contains a very useful method, [`.getDay()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay). It returns the index of the day this date falls on. If today is a Sunday, in your code `todaysDate.getDay()` will return 0, whilst if it was a Friday, it would return 5. Great! 🌟 
1. We just need to see what day it is going to be `daysInTheFuture` days from today. The days of the week repeat after 7 days. So let us travel into the future, by writing `const futureDayOfTheWeek = todaysDate.getDay() + daysInTheFuture`. However, this number might not be between 0 and 6. (7 values) To do that, we can use the [remainder operator `%`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder). Then, the result of `futureDayOfTheWeek % 7` is the exact index of the day you need, so just return the corresponding day name! 😊 🤟